.github/ISSUE_TEMPLATE/Standard-Dependencies-Issue-Template.md

```diff
@@ -0,0 +1,54 @@
+---
+name:  Dependencies Issue
+about: Use this template for all Dependencies Issues, even those automatically generated
+labels: dependencies
+---
+
+# Description
+- Information about the dependency and its role in the project
+- Current version - what are we running
+- Target Version - what are we changing it to
+- DependaBot PR link if available
+
+## Features and fixes available
+- List of important bug fixes, features or security patches
+- List of the important changes in this version that could affect the workflow
+
+## Additional information
+- Additional information about the version update.
+- Guideline that can be followed for the process
+
+## QA Needs
+< Indicate whether this item needs QA testing. For those that do, indicate any special setup or tool set the QA team might need. >
+
+## Acceptance Criteria
+< list of behaviors that must be present in order to satisfy the required functionality described above. >
+- [ ] Dependency updated in the build
+- [ ] Unit tests passing
+
+## Announcement Message
+devchat: < Dependency name > has been updated to <dependency target version> See <link to new features or this Issue > for available features and fixes
+
+## Progress
+### Ready for Planning
+- [ ] Story and Business Value complete
+- [ ] Initial Acceptance Criteria Defined
+- [ ] The person who will Approve story identified
+- [ ] Proposed Solution provided
+
+### Ready for Review
+- [ ] Testing implemented and passed
```

Comment:
Testing shouldn't be necessary for a dependency update

Reply:
That's Unit and Integration tests  for that checkbox. Don't you have to run them before you go to Review? Maybe we need to be more specific in all the templates?

Reply:
Can we leave it as is? The testing is necessary but the implementation of new/modified tests should not be. Unless of course something fails. But with the new unit testing, we may decide to force creation of new unit tests...